lib/benchee/conversion/count.ex

```diff
@@ -76,7 +84,7 @@ defmodule Benchee.Conversion.Count do
   ## Examples
 
       iex> Benchee.Conversion.Count.scale(12345, :one)
-      12345
+      12345.0
```

Comment:
Potential issue: even when not scaling (i.e. scaling to the base unit), we are dividing. I guess that would only change integers to floats, since the base unit should always be 1